src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js

```diff
@@ -358,6 +367,7 @@ const FishBeltTransectInputs = ({
             sizeBinValidationProperties,
             'size_bin',
           )}
+          hasObservationRecords={hasFishBeltObservations}
           value={formik.values.size_bin}
```

Comment:
you could make this something like `additionalText={hasFishbeltObservations ? language.whatever.whatever : null}`

You can think of this suggestion as inverting control logic to the parent instead of the child.